src/inmanta/data/__init__.py

```diff
@@ -4796,27 +4826,38 @@ async def get_resources_for_version_raw_with_persistent_state(
         version: int,
         projection: Optional[list[str]],
         projection_presistent: Optional[list[str]],
+        project_attributes: Optional[list[str]] = None,
```

Comment:
I added this so we don't have to pull in all attributes for the resource when not needed, this can be a  LOT of data